features/onboarding/impl/src/main/res/values-sk/translations.xml

```diff
@@ -4,6 +4,6 @@
   <string name="screen_onboarding_sign_in_with_qr_code">"Prihlásiť sa pomocou QR kódu"</string>
   <string name="screen_onboarding_sign_up">"Vytvoriť účet"</string>
   <string name="screen_onboarding_welcome_message">"Vitajte v najrýchlejšom %1$s vôbec. Nadupaný pre rýchlosť a jednoduchosť."</string>
```

Comment:
Looks like we are missing 1 sentence to be translated here. For all the languages above.
```
screen_onboarding_welcome_message
```